recipe_scrapers/practicalselfreliance.py

```diff
@@ -0,0 +1,28 @@
+from ._abstract import AbstractScraper
+
+
+class PracticalSelfReliance(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "practicalselfreliance.com"
+
+    def title(self):
+        return self.schema.title()
+
+    def total_time(self):
+        return self.schema.total_time()
+
+    def yields(self):
+        return self.schema.yields()
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        return self.schema.ingredients()
+
+    def instructions(self):
+        return self.schema.instructions()
+
+    def ratings(self):
```

Comment:
It looks like you forgot to include the test for `ratings` and I think adding it will fix the issue with coverage. Happy to approve it otherwise.

Reply:
updated to remove ratings method that's not actually available on this site